contracts/Safe.sol

```diff
@@ -17,8 +17,8 @@ import "./external/SafeMath.sol";
 /// @author Stefan George - <stefan@gnosis.io>
 /// @author Richard Meissner - <richard@gnosis.io>
 contract Safe is
-    EtherPaymentFallback,
     Singleton,
+    EtherPaymentFallback,
```

Comment:
Check singleton contract docs to understand this change 😄

Reply:
Please link it or add further explanining

Reply:
Removed it from the PR and created a separate one: https://github.com/safe-global/safe-contracts/pull/513